shelley/chain-and-ledger/shelley-spec-ledger-test/test/Test/Shelley/Spec/Ledger/Examples/PoolLifetime.hs

```diff
@@ -16,7 +16,8 @@ module Test.Shelley.Spec.Ledger.Examples.PoolLifetime
 where
 
 import Cardano.Ledger.Era (Crypto (..))
-import Cardano.Ledger.Val (invert, (<+>), (<->), (<×>))
+import Cardano.Ledger.Val ((<+>), (<->), (<×>))
+import Data.Group (invert)
```

Comment:
This change seems somewhat in the opposite direction from the changes above, where you moved `invert` to _not_ be the `Data.Group` version but defined it separately in `Cardano.Ledger.Val`

Reply:
In this file. The programmers use invert on an object that is not in the Val class. In the old code one could import Value to get the Group member of invert. That seems a bit perverse. This makes it very clear.

Reply:
In this file they use invert on DiffCoin, which is not in the Val class. It is clear they want the group invert.  In the previous code it was happenstance that the two were the same function.